src/pages/routes/ui/Routing.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { ProfilePage } from "pages/profile-page/ui";
```

Comment:
we don't know about ui and other segments from here

Reply:
import now looks like "pages/profile-page"